upgrading/automated_upgrades.adoc

```diff
@@ -525,6 +530,17 @@ link:http://docs.ansible.com/ansible/playbooks_delegation.html#delegation[`deleg
 If your {product-title} cluster uses a mixed environment or gcePD storage,
 you need to take more steps before you upgrade it.
 
+[[special-considerations-for-large-scale-upgrades]]
+==== Special considerations for large-scale upgrades
+
+For large-scale cluster upgrades, which involve at least 10 worker nodes and
+thousands of projects and pods, pre- and post- storage migration must be
```

Comment:
Can you clarify "pre- and post- storage migration?"

Reply:
@yocum137 Any suggestions on the wording here?

Reply:
How about this:

"... thousands of projects and pods, the API object storage migration should be performed prior to running the upgrade playbooks, and then again after the upgrade has successfully completed. Otherwise, the upgrade..."